style.css

```diff
@@ -1,3 +1,72 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
 * {
   box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-bar {
```

Comment:
- [ ] Kyndly stick to the figma design as much as possible

- Please add more space between the nav-bar and the main content 

![image](https://user-images.githubusercontent.com/95243996/191749540-0f6ce9b7-86af-4afc-89bc-17fca4fb2529.png)